### PR TITLE
Adding v1 of the snowflake fail open CFT and instructions

### PR DESCRIPTION
**Pull request by salgattcy**
## Description of the change

> Adding in V1 of the snowflake fail open support

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Jira issue referenced in commit message and/or PR title

### CODE REVIEW

File: snowflake/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+| SSHKeyName            | This should be the name of the SSH Key that should be assigned to the created instances                                                                                                                                                                                                                                            | AWS_SSH_Key                                                   |
+| SideCarDomainName     | This should be the high level domain name associated with the sidecar. This will be used to create a reverse proxy entry of the form SnowFlakeAccountName.SideCarDomainName that reverse proxies to RepoHost                                                                                                                       | aws.mydomain.us                                               |
```
Comment (ha2398) [inline snowflake/README.md:61]: To be consistent with other occurrences, I think this should read `SidecarDomainName`

File: snowflake/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| SidecarNamePrefix     | This parameter is used to identify the elements of this stack and also the events created during runtime and should match the prefix of the sidecar                                                                                                                                                                                | cyral-xyz123                                                  |
+| SnowFlakeAccountName  | This should just be the account name for the customer’s snowflake instance                                                                                                                                                                                                                                                         | xx#####                                                       |
```
Comment (ha2398) [inline snowflake/README.md:64]: How about using `SnowflakeAccountName` instead?

File: snowflake/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+3. Once the certificate is validated, open the AWS EC2 console
+4. Locate the Cyral sidecar load balancer that hosts your snowflake instance(s)
+5. Click the `view/edit certificates` next to the 443 Listener on the load balancer
+6. Click the `+` sign at the top of the page to add another certificate
+7. Locate the certificate created in step 2 and place a check next to it
+8. Click the `Add` button to add the certificate 
+
+# Failing Open or Closed
+
+##Failing Open
```
Comment (ha2398) [inline snowflake/README.md:86]:
```suggestion
## Failing Open
```

File: snowflake/cft_snowflake_failopen.yaml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      Type: AWS::AutoScaling::LaunchConfiguration
+      Properties:
+        KeyName:
+          Ref: SSHKeyName
+        ImageId: !Ref AmiId
+        SecurityGroups:
+          Ref: "InstanceSecurityGroup"
+        InstanceType:
+          Ref: "InstanceType"
+#        LaunchConfigurationName: !Join ['-', [!Ref SidecarNamePrefix, SnowflakeFailOpen]]
```
Comment (ha2398) [inline snowflake/cft_snowflake_failopen.yaml:68]: Nit: remove commented out code.

**Review by ha2398 — APPROVED** [review]

**Review by wcmjunior — APPROVED** [review]

**Pull request merged**